specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-12-01/Marketplace.json

```diff
@@ -0,0 +1,1068 @@
+          "description": "Gets request date",
+          "type": "string",
+          "readOnly": true
+        },
+        "justification": {
+          "description": "Gets justification",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RequestApprovalsList": {
+      "description": "List of admin request approval resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RequestApprovalResource"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next set of notifications list results if there are any.",
+          "type": "string"
+        }
+      }
+    },
+    "RequestApprovalResource": {
+      "description": "Request approval resource.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/RequestApprovalProperties",
+          "description": "The privateStore approval request data structure.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "RequestApprovalProperties": {
+      "description": "Approval request resource properties",
+      "type": "object",
+      "properties": {
+        "offerId": {
+          "description": "Gets or sets unique offer id.",
+          "type": "string"
+        },
+        "offerDisplayName": {
+          "description": "Gets or sets offer display name",
+          "type": "string",
+          "readOnly": true
+        },
+        "publisherId": {
+          "description": "The offer's publisher id",
+          "type": "string"
+        },
+        "plansDetails": {
+          "description": "Gets or sets the plans details",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/planDetails"
+          }
+        },
+        "isClosed": {
+          "description": "Gets or sets a value indicating whether the request is closed",
+          "type": "boolean",
+          "readOnly": true
+        },
+        "messageCode": {
+          "description": "Gets or sets the request approval message code",
+          "type": "integer",
+          "format": "int64"
+        }
+      }
+    },
+    "planDetails": {
+      "description": "Return plan with request details",
+      "type": "object",
+      "properties": {
+        "planId": {
+          "description": "Gets or sets Plan Id ",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the plan status",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Pending",
+            "Rejected",
+            "Approved",
+            "None"
+          ],
+          "x-ms-enum": {
+            "name": "status",
+            "modelAsString": true
+          }
+        },
+        "requestDate": {
```

Comment:
mark this as datetime -> "format": "date-time"

Reply:
done